react/modules/isGoCommerceAccount.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isGoCommerceAccount = (data: string | any): boolean => {
+  let account = ''
+  if (typeof data === 'string') {
```

Comment:
I would rather wait for this:
https://github.com/vtex-apps/render-runtime/pull/408

Reply:
@klzns  I would rather release this as it is, since it saves some 60kb + parse time, and update it later

Reply:
:developers: